test/functional/feature_nulldummy.py

```diff
@@ -46,6 +46,7 @@ def set_test_params(self):
             '-segwitheight=432',
```

Comment:
An alternative would be to subtract 300 from this constant (and all dependent constants)?

Reply:
Great idea--bigger diff, but resolves the issue (locally for me, at least) and speeds up the test from 4 to 2.5 seconds

Reply:
It looks like we can reduce the blocks from 432 to 104